typings/rendition.d.ts

```diff
@@ -346,6 +346,13 @@ declare module 'rendition' {
 
 	class Search extends RenderableElementWithProps<SearchProps, any> {}
 
+	class Toggle extends RenderableElementWithProps<
+		DefaultProps & {
+			brand: JSX.Element;
```

Comment:
What is the `brand` property? It seems to be completely unused

Reply:
You right not used, stupid copy/paste ...